pydata_sphinx_theme/static/css/theme.css

```diff
@@ -0,0 +1,85 @@
+/*****************************************************************************
+ * Custom font
+**/
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+:root {
+  /*****************************************************************************
+   * Bootstrap variables are availlable:
+   * https://getbootstrap.com/docs/4.0/getting-started/theming/#css-variables
+  **/
+
+  /*****************************************************************************
+  * Font size
+  **/
+  --fs-base: 15px; /* base font size - applied at body / html level */
+
+  /* heading font sizes */
+  --fs-h1: 36px;
+  --fs-h2: 32px;
+  --fs-h3: 26px;
+  --fs-h4: 21px;
+  --fs-h5: 28px;
+  --fs-h6: 18px;
+
+  /* smaller then heading font sizes*/
+  --fs-milli: 12px;
+
+  /*****************************************************************************
+  * Font family
+  **/
+  --ff-base: 'Lato', sans-serif;
+  --ff-heading: 'Open Sans', sans-serif;
+
+  /*****************************************************************************
+  * Color
+  **/
+  --color-primary: #130654;
+  --color-text-base: #333;
+  --color-h1: var(--color-primary);
+  --color-h2: var(--color-primary);
+  --color-h3: var(--color-text-base);
+  --color-h4: var(--color-text-base);
+  --color-h5: var(--color-text-base);
+  --color-h6: var(--color-text-base);
+  --color-paragraph: var(--color-text-base);
+  --color-link: #005b81;
+  --color-link-hover: #e32e00;
+  --color-headerlink: #c60f0f;
+  --color-headerlink-hover: white;
+  --color-preformatted-text: #222;
+  --color-preformatted-background: #fafafa;
+
+  /* admonition colors */
+  --color-admonition-base: var(--blue);
+  --color-admonition-attention: var(--orange);
+  --color-admonition-attention-light: #ffedcc;
```

Comment:
I am not sure we need to necessarily add variables for all of those admonition types (we can always add later if there is demand for it). Maybe we could keep it on the 4 colors we use now.

What I was originally thinking for those admonitions is to reuse the bootstrap `success`, `danger`, `warning`, `info` variables. 
The problem though is that we also need the "-light" version, and that doesn't seem fully straightforward. Eg I found https://codyhouse.co/blog/post/how-to-combine-sass-color-functions-and-css-variables, that has a way to allow something like `lightness(var(--color-primary), 1.2)`

Reply:
Limit the admonition types is fine for me, semantics in css are not that important I think.

Regarding the `*-light` variant, this possible for sure, it could be looked upon as a clear theme choice.

Reply:
I diggel into the sass color functions combined with css variables. All solutions available start from defining the colors either based in RGB (e.g. red: `255, 0, 0`) or HSL (e.g red: `0%, 100%, 50%`.

With RBG, it is possible to lighten a color by adding an alpha channel. Darkening the color is harder I think.
HSL provides more flexibility.

The main disadvantage is the theme colors MUST be defined in either RGB, HSL and never Hex. Somehow this feels like a downside for the theme user who quickly wants to define his/her theme colors on top?

@jorisvandenbossche @choldgraf do you have thoughts on this?